the-path-to-1-0/index.html

```diff
@@ -235,7 +235,7 @@ <h1 class="entry-title">the path to 1.0</h1>
 <p>As you may have noticed, we release a 0.9.x update with bug fixes and new features almost daily, so why haven&rsquo;t we released 1.0 yet?</p>
 <h2>Shifting from &ldquo;a bag of functions&rdquo; to a proper module</h2>
 <p>We love new functionality, of course, but we must take a step back from new development and consolidate what we have in a properly usable module.</p>
-<p>Most of the contributors are damn good SQL Server Pros, a few are damn good PowerShell programmers, with the majority of &ldquo;casual&rdquo; developers being able to successfully publish a new function in less than a week time. But what we aren&rsquo;t so good at is advertising our needs. We recently updated our&nbsp;<a href="https://github.com/dataplat/dbatools/blob/master/contributing.md">contributing.md</a>&nbsp;to be more on par with the latest months of developing. Stay tuned there, as the few TODOs left are coming, but the path is mostly set.</p>
+<p>Most of the contributors are damn good SQL Server Pros, a few are damn good PowerShell programmers, with the majority of &ldquo;casual&rdquo; developers being able to successfully publish a new function in less than a week time. But what we aren&rsquo;t so good at is advertising our needs. We recently updated our&nbsp;<a href="https://github.com/dataplat/dbatools/blob/master/CONTRIBUTING.md">contributing.md</a>&nbsp;to be more on par with the latest months of developing. Stay tuned there, as the few TODOs left are coming, but the path is mostly set.</p>
```

Comment:
```suggestion
<p>Most of the contributors are damn good SQL Server Pros, a few are damn good PowerShell programmers, with the majority of &ldquo;casual&rdquo; developers being able to successfully publish a new function in less than a week time. But what we aren&rsquo;t so good at is advertising our needs. We recently updated our&nbsp;<a href="https://github.com/dataplat/dbatools/blob/development/CONTRIBUTING.md">contributing.md</a>&nbsp;to be more on par with the latest months of developing. Stay tuned there, as the few TODOs left are coming, but the path is mostly set.</p>
```

Please set the branch to this one to be development as well.